documentation/specs/DisplayTable.md

```diff
@@ -0,0 +1,238 @@
+# `DisplayTable` Component Specification
+
+## Overview
+
+A `DisplayTable` is used for presenting information displayed across columns and rows.
+
+---
+
+## Design
+
+`DisplayTable` is for presentational concerns only. It doesn't support interactions such as selection, sorting, and row expansion. See the `Table` component for an interactive table accommodating advanced use cases.
+
+Without interaction, complexities are shifted to surfacing a simple component API, structuring the table for style requirements, and ensuring appropriate application of accessibility markup.
+
+`DisplayTable` will use React Aria's `useTable` under the hood to reliably generate accessibile markup for the table.
+
+`DisplayTable` will use `columns` and `rows` as the primary mechanism for passing data to the table. While this is a slight departure from our dot-notation pattern used throughout the rest of Easy UI, this makes for a consistent data input pattern with our `Table`-related components.
+
+### API
+
+```ts
+type DisplayTableProps = AriaLabelingProps & {
+  /** Columns for the table. */
+  columns: Column[];
+
+  /**
+   * Horizontal alignment of data within their cells.
+   * @default start
+   */
+  dataAlignment?: "start" | "center" | "end";
+
+  /**
+   * Marks the table has having no row headers. For accessibility reasons, this should be avoided when possible.
+   */
+  hasNoRowHeaders?: boolean;
```

Comment:
instead of checking for a row header through a `heading` property in the first cell of the row data, control it as a prop in the `DisplayTable` component, since it should be table-wide setting instead of a cell-specific setting (either all or no rows should have row headers).

name is negated on purpose to discourage not having row headers. open to feedback on a better name.

Reply:
I think this makes sense! One of the examples uses `hasRowHeaders` though ;)

Reply:
whoops good catch